doc/release-notes.md

```diff
@@ -31,15 +31,190 @@ Downgrading to a version older than v19.2.0 is not supported due to changes
+    - **Limitations and Considerations**
+      - **Compatibility:** Encrypted connections may only be established between nodes that have BIP324 enabled. Nodes without encryption support will continue to communicate over unencrypted channels.
+    - **Status and Future Plans**
+      - **Experimental Phase:** This feature is currently in an experimental phase. Users are encouraged to test and provide feedback to ensure stability.
+      - **Future Development:** BIP324 will likely be enabled by default in the next minor version, v22.1.0.
+
+
+## Devnet Breaking Changes
+
+- **Hardfork Activation Changes**
+    - `BRR` (`realloc`), `DIP0020`, `DIP0024`, `V19`, `V20`, and `MN_R` hardforks are now activated at **block 2** instead of block **300** on devnets.
+    - **Implications:**
+        - Breaking change.
+        - Inability to sync on devnets created with earlier Dash Core versions and vice versa.
+
+## Tests
+
+- **Regtest Network Softfork Activation Heights**
+    - For the `regtest` network, the activation heights of several softforks have been set to **block height 1**.
+    - **Customization:** Can be changed using the runtime setting `-testactivationheight=name@height`.
+    - *(dash#6214)*
+
+## Statistics
+
+### New Features
+
+- **Statsd Client Enhancements**
+    - Supports queueing and batching messages.
+        - **Benefits:**
+            - Reduces the number of packets.
+            - Lowers the rate at which messages are sent to the Statsd daemon.
+
+- **Batch Configuration Options**
+    - **Maximum Batch Size:**
+        - Adjustable using `-statsbatchsize` (in bytes).
+        - **Default:** **1KiB**.
+    - **Batch Send Frequency:**
+        - Adjustable using `-statsduration` (in milliseconds).
+        - **Default:** **1 second**.
+        - **Note:** `-statsduration` does not affect `-statsperiod`, which dictates how frequently some stats are *collected*.
+
+### Deprecations
+
+- **Deprecation of `-platform-user`**
+    - `-platform-user` is deprecated in favor of the whitelist feature.
+    - In releases **22.x** of Dash Core, it has been renamed to `-deprecated-platform-user`.
+    - It will be removed in version **23.x**.
+
+- **`-statsenabled` Deprecation**
+    - Now implied by the presence of `-statshost`.
+    - It will be removed in version **23.x**.
+
+- **`-statshostname` Replacement**
+    - Deprecated and replaced with `-statssuffix` for better representation of the argument's purpose.
+    - **Behavior:** Behave identically to each other.
+    - It will be removed in version **23.x**.
+
+- **`-statsns` Replacement**
+    - Deprecated and replaced with `-statsprefix` for better representation of the argument's purpose.
+    - **Behavior:** `-statsprefix` enforces the usage of a delimiter between the prefix and key.
+    - It will be removed in version **23.x**.
+
+## GUI Changes
```

Comment:
consider including some minor changes to release notes:
    Merge #6444: fix: add platform transfer to "most common" filter
    Merge bitcoin-core/gui#740: Show own outputs on PSBT signing window
    Merge bitcoin-core/gui#365: Draw "eye" sign at the beginning of watch-only addresses
    69a1305978a65978feae17e1a5715b4b70f82e91 Merge bitcoin-core/gui#309: Add access to the Peers tab from the network icon (Hennadii Stepanov)
    Merge bitcoin-core/gui#256: Save/restore column sizes of the tables in the Peers tab

Reply:
Any recommended text?